src/helpers.js

```diff
@@ -24,37 +24,75 @@ const promisify = require('@google-cloud/common').util.promisify;
 const gax = require('google-gax');
 
 /*!
- * Find a given image and fire a callback with the appropriate image structure.
+ * Convert non-object request forms into a correctly-formatted object.
  *
- * @param {object} image An object representing what is known about the
- *   image.
+ * @param {object|string|Buffer} request An object representing an
+ *   AnnotateImageRequest. May also be a string representing the path
+ *   (filename or URL) to the image, or a buffer representing the image itself.
+ *
+ * @returns An object representing an AnnotateImageRequest.
+ */
+let _requestToObject = request => {
+  if (is.string(request) === true) {
```

Comment:
s/ `if (is.string(request) === true) {` / `if (is.string(request)) {` /

Reply:
Changed.